cmd/git-sync/main.go

```diff
@@ -600,7 +601,8 @@ func main() {
 			}
 		}
 
-		if changed, hash, err := git.SyncRepo(ctx); err != nil {
+		changed, hash, err := git.SyncRepo(ctx)
+		if err != nil {
```

Comment:
Why this stylistic change?

Reply:
It's a mistake, sorry. I'll change it back.

I thought "hash" wouldn't be available in the "else if" scope if it was declared inline in the "if" but I now see that "changed" was initialized/available just fine, so I think I may not have been _sufficiently caffeinated_ to see it properly. I'll change it back.

I'm in the habit that in Golang when you make a var inside an "if" or a "for" it is scoped to that block and I mistook the inline "if" var init as being scoped inside its "if " block when apparently it's scoped just outside of it. My bad.